### ISSUE TITLE
Invalid dependency in pom

### ISSUE THREAD

**Issue by fafk** [netopyr/reduxfx#70]
Hi, I tried to add the reduxfx dependency to a project as described in the README, however the fontsawesome package cannot be downloaded via maven anymore.

[ERROR] Failed to execute goal on project wallet: Could not resolve dependencies for project xxx.jar:0.0.1-SNAPSHOT: Failed to collect dependencies at com.netopyr.reduxfx:reduxfx-all:pom:0.4.1 -> com.netopyr.reduxfx:reduxfx-fontawesomefx:jar:0.4.1 -> de.jensd:fontawesomefx-commons:jar:8.15: Failed to read artifact descriptor for de.jensd:fontawesomefx-commons:jar:8.15: Could not transfer artifact de.jensd:fontawesomefx-commons:pom:8.15 from/to bintray (http://jcenter.bintray.com): Access denied to: http://jcenter.bintray.com/de/jensd/fontawesomefx-commons/8.15/fontawesomefx-commons-8.15.pom -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException

Basically jcenter started enforcing HTTPS and therefore one gets 403 when trying to download it.

**Comment by manuel-mauky** [comment]
What could be done on the project side to fix this? It sounds like a tool problem with maven. Why doesn't it use https?

**Comment by fafk** [comment]
I guess it might be possible to tune it in settings.xml, but it's probably not good that it's failing by default with a vanilla maven (3.6.0). Other packages work fine. *shrugs*

https://jfrog.com/blog/secure-jcenter-with-https/
